src/components/PopoverWithMeasuredContent.js

```diff
@@ -62,7 +64,15 @@ class PopoverWithMeasuredContent extends Component {
     }
 
     shouldComponentUpdate(nextProps, nextState) {
-        return !_.isEqual(this.props, nextProps) || !_.isEqual(this.state, nextState);
+        if (this.props.isVisible
+            && (nextProps.windowWidth !== this.props.windowWidth
+                || nextProps.windowHeight !== this.props.windowHeight)) {
```

Comment:
I aligned it but not sure what should be the correct alignment here. Please suggest.

Reply:
So that the || is also aligned: ex.

```
&& (nextProps.windowWidth !== this.props.windowWidth
|| nextProps.windowHeight !== this.props.windowHeight)) {
```